mbf_abstract_nav/include/mbf_abstract_nav/move_base_action.h

```diff
@@ -166,6 +166,7 @@ class MoveBaseAction
 
   //! Replanning thread, running permanently
   boost::thread replanning_thread_;
+  bool replanning_thread_shutdown_{ false };
```

Comment:
This is C++11,,,  we didn't agree yet on bumping the standard, right?

Reply:
@spuetz, @RainerKuemmerle, unless you raise any concern, we will move master to C++17, setting the standard on the metapackage

Reply:
I am changing following the file convention since this is not required here